### PR TITLE
WE DID IT!

### PR DESCRIPTION
**Pull request by SamGetlan**

### CODE REVIEW

**Review by ykent4 — COMMENTED** [review]
Code looks good to me. We will need to make some adjustments to CSS and add the headbar picture

**Comment by gregoryew** [comment]
Looks good.  We need to place the header on the top and adjust CSS

File: welp/proxy/server.js [hunk 1]
```diff
@@ -5,8 +5,58 @@ const app = express();
...
 app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, 'public')));
+// app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, '/templates/services')));
+
+
+const clientBundles = './public/services';
+const serverBundles = './templates/services';
+const serviceConfig = require('./service-config.json');
+// const services = require('./loader.js')(clientBundles, serverBundles, serviceConfig);
```
Comment (AndrewPomo) [inline welp/proxy/server.js:15]: leftover comment?

File: welp/proxy/server.js [hunk 2]
```diff
@@ -5,8 +5,58 @@ const app = express();
...
+console.log('reviews >>> ', Reviews);
+
+const React = require('react');
+const ReactDom = require('react-dom/server');
+const Layout = require('./templates/layout');
+const App = require('./templates/app');
+const Scripts = require('./templates/scripts');
+
+const renderComponents = (components, props = {}) => {
+  // console.log('components -->', components);
```
Comment (AndrewPomo) [inline welp/proxy/server.js:42]: commented console.log

File: welp/proxy/server.js (hunk 1 above)
Comment (SamGetlan) [reply welp/proxy/server.js:15]: Thanks for the heads up

**Pull request merged**